charts/sn-platform/templates/presto/presto-serviceentry.yaml

```diff
@@ -0,0 +1,60 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+{{- if and .Values.components.sql_worker .Values.istio.enabled }}
+{{- range $i, $e := until (int .Values.presto.coordinator.replicaCount ) }}
+apiVersion: networking.istio.io/v1beta1
+kind: ServiceEntry
+metadata:
+  name: "{{ template "presto.coordinator" $ }}-{{ $i }}"
+spec:
+  hosts:
+  - "{{ template "presto.coordinator" $ }}-{{ $i }}.{{ template "presto.service" $ }}.{{ template "pulsar.namespace"  $ }}.svc.cluster.local"
+  location: MESH_INTERNAL
+  ports:
+  {{- if $.Values.tls.presto.enabled }}
+  - name: tls-web
+    number: {{ $.Values.presto.coordinator.ports.https }}
+    protocol: HTTPS
+  {{- else }}
+  - name: tcp-web
+    number: {{ $.Values.presto.coordinator.ports.http }}
+    protocol: HTTP
+  {{- end }}
+  resolution: DNS
+---
+{{- end }}
+{{- range $i, $e := until (int .Values.presto.worker.replicaCount) }}
```

Comment:
I think we can remove the unused `$e` variable here to simplify as below:

```
{{- range $i := until (int .Values.presto.coordinator.replicaCount ) }}
```

Reply:
applied, please review again